WordPressKit/WordPressKit/ServiceRequest.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+
+/// Enumeration to identify a service action
+///
+/// - subscribe: The action 'new'
+/// - unsubscribe: The action 'delete'
+/// - update: The action 'update'
```

Comment:
Nipticky: Documenting each case might not really be needed (`case subscribe = 'new'`)

Reply:
+1